app/src/main/java/com/lagradost/cloudstream3/AcraApplication.kt

```diff
@@ -37,11 +38,17 @@ import kotlin.concurrent.thread
 import kotlin.system.exitProcess
 
 class CustomReportSender : ReportSender {
-    // Sends all your crashes to google forms
+    // Sends crash logs to Google forms, Might give a false positive of having a Google Tracker in App Scans
     override fun send(context: Context, errorContent: CrashReportData) {
-        println("Sending report")
-        val url =
-            "https://docs.google.com/forms/d/e/1FAIpQLSfO4r353BJ79TTY_-t5KWSIJT2xfqcQWY81xjAA1-1N0U2eSg/formResponse"
+
+        println("Sending Crash Report")
+
+        val url = if (SettingsFragment.isTrueTvSettings()) {
+            "FORM_URL_FOR_TVs"
+        } else {
+            "FORM_URL_FOR_PHONES"
+        }
+
         val data = mapOf(
             "entry.1993829403" to errorContent.toJSON()
```

Comment:
have you checked that this key does not change for different google forms?

Reply:
how can i do that ?
probably no if created by same gmail account 🤔

Reply:
> have you checked that this key does not differ between forms?

hmm, found out that it won't be an issue because the form and its input data structure is same its just we have seperated their endpoint reciever url